Report/sections/2_design/language_design.tex

```diff
@@ -1,4 +1,14 @@
 \chapter{Language design}
+This chapter details the design of the language \fxfatal{INSERT LANGUAGE NAME HERE}.
+The language is intended to be a transitional language from visual programming to high-level language. Based on the problem analysis, it has been possible to specify the design for the user so that it hits the target group.
```

Comment:
'Hits the target group' could be reformulated

Reply:
Done